index.html

```diff
@@ -137,51 +138,8 @@ <h2>Why us</h2>
             <h2>Our courses</h2>
             <hr>
         </div>
-        <div class="card-container">
-            <div class="course-card">
-                <div class="card-body">
-                    <img width="200" height="200" src="assets/images/kid6.jpg" alt=" about us image">
-                    <h3>Scratch Game Development</h3>
-                    <p class="content">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Pariatur soluta, cum sequi, nobis nisi odit molestiae a voluptatum quasi, deleniti hic dignissimos labore accusantium obcaecati iure in eaque enim ut!. </p>
-                </div>
-                <a href="#" class="btn btn-view-more"> Read more <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-arrow-right-short" viewBox="0 0 16 16">
-                    <path fill-rule="evenodd" d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"/>
-                    </svg>
```

Comment:
- [ ] Please ensure to stick to the design guideline as much as possible. Your pages should look almost identical to the original design, outside the content differences.
- [ ] The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.
- [ ] The pages use the same fonts or similar fonts from the original design.
- [ ] All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158699041-9b9cac87-b8d9-4104-ae60-8f706ef25667.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158699141-0880f477-65c2-405d-9e38-c13d4a417f56.png)